src/Sentry/Platforms/Native/buildTransitive/Sentry.Native.targets

```diff
@@ -7,21 +7,30 @@
 
   Note:Target framework conditions should be kept synchronized with src/Sentry/buildTransitive/Sentry.Native.targets -->
 <Project>
-  <ItemGroup Condition="$(TargetFramework.StartsWith('net8')) and ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'WinExe') And '$(RuntimeIdentifier)' == 'win-x64'">
+  <ItemGroup Condition="
+      ($(TargetFramework.StartsWith('net8')) Or $(TargetFramework.StartsWith('net9')))
+      and ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'WinExe')
+      And '$(RuntimeIdentifier)' == 'win-x64'">
     <DirectPInvoke Include="sentry-native" />
     <NativeLibrary Include="$(MSBuildThisFileDirectory)..\sentry-native\win-x64\sentry-native.lib" />
     <NativeLibrary Include="dbghelp.lib" />
     <NativeLibrary Include="winhttp.lib" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(TargetFramework.StartsWith('net8')) and ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'WinExe') And '$(RuntimeIdentifier)' == 'linux-x64'">
+  <ItemGroup Condition="
+      ($(TargetFramework.StartsWith('net8')) Or $(TargetFramework.StartsWith('net9')))
+      and ('$(OutputType)' == 'Exe' Or '$(OutputType)' == 'WinExe')
+      And '$(RuntimeIdentifier)' == 'linux-x64'">
```

Comment:
@bruno-garcia [commented here](https://github.com/getsentry/sentry-dotnet/pull/3513#discussion_r1800237160):
> maybe we should invert the logic. If !StartsWith("net4") && !Contains("netstandard")
So when we add new netX.0 it includes it here automatically?

Reply:
Good call.